php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/CheckContext.java

```diff
@@ -117,13 +117,19 @@
    */
   FileIssue newFileIssue(PHPCheck check, String message);
 
+  List<PhpIssue> getIssues();
+
+  SymbolTable symbolTable();
+
   /**
    * @return the current file
+   * @deprecated since 2.10. Use {@link CheckContext#getPhpFile()}
    */
+  @Deprecated
   File file();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)